libs/community/langchain_community/chat_loaders/__init__.py

```diff
@@ -17,3 +17,34 @@
     ChatSession
 
 """  # noqa: E501
+
+import importlib
+from typing import Any
+
+_module_lookup = {
+    "BaseChatLoader": "langchain_community.chat_loaders.base",
+    "FolderFacebookMessengerChatLoader": "langchain_community.chat_loaders.facebook_messenger",  # noqa: E501
+    "GMailLoader": "langchain_community.chat_loaders.gmail",
+    "IMessageChatLoader": "langchain_community.chat_loaders.imessage",
+    "LangSmithDatasetChatLoader": "langchain_community.chat_loaders.langsmith",
+    "LangSmithRunChatLoader": "langchain_community.chat_loaders.langsmith",
+    "SingleFileFacebookMessengerChatLoader": "langchain_community.chat_loaders.facebook_messenger",  # noqa: E501
+    "SlackChatLoader": "langchain_community.chat_loaders.slack",
+    "TelegramChatLoader": "langchain_community.chat_loaders.telegram",
+    "WhatsAppChatLoader": "langchain_community.chat_loaders.whatsapp",
+    "map_ai_messages": "langchain_community.chat_loaders.utils",
+    "map_ai_messages_in_session": "langchain_community.chat_loaders.utils",
+    "merge_chat_runs": "langchain_community.chat_loaders.utils",
+    "merge_chat_runs_in_session": "langchain_community.chat_loaders.utils",
+    "nanoseconds_from_2001_to_datetime": "langchain_community.chat_loaders.imessage",
```

Comment:
```suggestion
```

Reply:
looks like a somewhat strange functionality for a chat loader namespace to declare as a public function. feels like internal functionality